packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -579,12 +580,17 @@ class SliverReorderableListState extends State<SliverReorderableList> with Ticke
     setState(() {
       if (_dragInfo != null) {
         cancelReorder();
+      } else if (_recognizer != null && _recognizerPointer != event.pointer) {
+        _recognizer!.dispose();
+        _recognizer = null;
+        _recognizerPointer = null;
```

Comment:
If a second gesture comes in here, wouldn't we want to also cancel the previous reorder? If we aren't going to cancel the current drag, then wouldn't it make more sense to ignore the new gesture and wait for the first one to complete instead of reseting the recognizer to the new one? They may be trying to drag different items.

Reply:
Fair enough. I missed that this is only valid for the case where _dragInfo is null. Sounds good. Thx.